### ISSUE TITLE
wiktionary-parser is missing

### ISSUE THREAD

**Issue by gyankos** [commonsense/conceptnet5#219]
Where can I find the wiktionary-parser that is required to convert custom wikipedia dumps into the json file format? I need some wikipedia languages that are not currently supported by your dump in Amazon S3. Thanks.

**Comment by rspeer** [comment]
https://github.com/LuminosoInsight/wikiparsec provides both wiktionary-parser and wiki2text.

You certainly won't be able to use a Wiktionary parser to parse Wikipedia, but maybe wiki2text is what you want.

**Comment by gyankos** [comment]
Edit: wiktionary dumps. 

Thanks, this information was not provided in the readme.

**Comment by rspeer** [comment]
Ah okay. I'll find the right place to document that.

Every Wiktionary has a different format of entries, and the Wiktionary parser so far is only designed for the English, French, and German ones. Which Wiktionary do you need?

(In case you are running into the usual confusion: every Wiktionary defines words in hundreds of languages. The difference between the projects is what language the _editors_ speak, and therefore what language the text content of the definitions is in.)

**Comment by gyankos** [comment]
Ok, I took a look at the project and I noticed that they use different parsers for different wikis. So, I'll use the parser just for the updated English version, and I'll check out whether there are LuminosoInsight/wikiparsec forks supporting other languages (e.g., Italian).

Actually, I think that  xy-wiktionary may not contain all the words for language yz. Therefore, I was only trying to get the words from their original language wiki (yz-wiktionary). Thanks.
